_posts/pandera-dataframe-validation.md

```diff
@@ -0,0 +1,263 @@
+---
+layout: single
+title: "Statistical Data Validation with Pandera"
+excerpt: "A flexible and expressive toolkit for performing statistical validation checks on pandas data structures."
+author: "Niels Bantilan"
+permalink: /posts/pandera-dataframe-validation
+header:
+    overlay_image: images/pandas.jpg
+    overlay_filter: 0.6
+categories:
+  - blog-post
+---
+
+<span style='font-size: 16px;'>
+    Photo credit:
+    <a href="https://www.flickr.com/photos/nationalzoo/5371290900/in/photostream/"
+       target="_blank">
+    Ann Batdorf, Smithsonian's National Zoo</a>
+</span>
+
+Modern data engineering and analysis workflows will often involve using data
```

Comment:
How about a line or two here that just sets up that pandera just went through the review process and was accepted by pyopensci! See this blog for an example. 

https://ropensci.org/blog/2019/11/05/tidync/